tests/rustdoc/item-desc-list-at-start.rs

```diff
@@ -0,0 +1,8 @@
+// @has item_desc_list_at_start/index.html
+// @count - '//ul[@class="item-table"]/li/div/li' 0
```

Comment:
Wouldn't it be better to also include a snapshot check as well? They are quite useful to be absolutely sure that nothing changed in a given location.